build.sbt

```diff
@@ -58,7 +58,7 @@ lazy val daffodil = project
 
 lazy val macroLib = Project("daffodil-macro-lib", file("daffodil-macro-lib"))
   .settings(commonSettings, nopublish)
-  .settings(libraryDependencies += "org.scala-lang" % "scala-reflect" % scalaVersion.value)
+  .settings(libraryDependencies += "org.scala-lang" % "scala-reflect" % "2.13.16")
```

Comment:
This seems a bit odd, will this cause issues if we always depend on 2.13.16 even though we are building scala 3? Should this only add scala-reflect if scalaVersion is 2.13.x? Similar to what you have in Dependencies.scala?

Reply:
So scala-reflect is only release up to scala 2.13, there is no version 3 hence hardcoding to 2.13.16 (the latest version at the time)

https://mvnrepository.com/artifact/org.scala-lang/scala-reflect